tests/Unit/Evolution/Systems/NewtonianEuler/BoundaryConditions/DirichletAnalytic.py

```diff
@@ -0,0 +1,263 @@
+                              _soln_adiabatic_index, _soln_perturbation_size)
+    return flux.momentum_density_flux_impl(
+        soln_momentum_density(face_mesh_velocity,
+                              outward_directed_normal_covector, coords, time,
+                              dim),
+        soln_energy_density(face_mesh_velocity,
+                            outward_directed_normal_covector, coords, time,
+                            dim), velocity, pressure)
+
+
+def soln_flux_energy_density(face_mesh_velocity,
+                             outward_directed_normal_covector, coords, time,
+                             dim):
+    velocity = hydro.spatial_velocity(coords, time, _soln_mean_velocity(dim),
+                                      _soln_wave_vector(dim), _soln_pressure,
+                                      _soln_adiabatic_index,
+                                      _soln_perturbation_size)
+    pressure = hydro.pressure(coords, time, _soln_mean_velocity(dim),
+                              _soln_wave_vector(dim), _soln_pressure,
+                              _soln_adiabatic_index, _soln_perturbation_size)
+    return flux.energy_density_flux_impl(
+        soln_momentum_density(face_mesh_velocity,
+                              outward_directed_normal_covector, coords, time,
+                              dim),
+        soln_energy_density(face_mesh_velocity,
+                            outward_directed_normal_covector, coords, time,
+                            dim), velocity, pressure)
+
+
+def soln_velocity(face_mesh_velocity, outward_directed_normal_covector, coords,
+                  time, dim):
+    return hydro.spatial_velocity(coords, time, _soln_mean_velocity(dim),
+                                  _soln_wave_vector(dim), _soln_pressure,
+                                  _soln_adiabatic_index,
+                                  _soln_perturbation_size)
+
+
+def soln_specific_internal_energy(face_mesh_velocity,
+                                  outward_directed_normal_covector, coords,
+                                  time, dim):
+    return hydro.specific_internal_energy(coords, time,
+                                          _soln_mean_velocity(dim),
+                                          _soln_wave_vector(dim),
+                                          _soln_pressure,
+                                          _soln_adiabatic_index,
+                                          _soln_perturbation_size)
+
+
+def data_error(face_mesh_velocity, outward_directed_normal_covector, coords,
+               time, dim):
+    return None
+
+
+_data_adiabatic_index = 1.4
+_data_strip_bimedian_height = 0.5
+_data_strip_thickness = 0.5
+_data_strip_density = 2.0
+_data_strip_velocity = 0.5
+_data_background_density = 1.0
+_data_background_velocity = -0.5
+_data_pressure = 2.5
+_data_perturb_amplitude = 0.1
+_data_perturb_width = 0.035355339059327376
```

Comment:
is it required that this be a weird non-ergonomic value? It appears to be an input in the places it's used, but maybe it's important to be a particular thing relative to other values?

Reply:
I expect not, I just took something from the documentation or wherever I found an example. I'll truncate to 0.03 and see if the tests still pass :)